tictactoe_env.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+
+
+class TicTacToeEnv(object):
+    def __init__(self):
+        """
+        state space: 2x3x3
+        action space: 18 possible values -- 1 hot
```

Comment:
I think the action space should be only 3x3.  In AlphaGo there is no color in the action space.